batchflow/models/torch/blocks.py

```diff
@@ -109,10 +99,8 @@ class ResBlock(nn.Module):
     kwargs : dict
         Other named arguments for the :class:`~.layers.ConvBlock`
     """
-    def __init__(self, inputs=None, layout='cnacna', filters='same', kernel_size=3, strides=1,
-                 downsample=False, bottleneck=False, se=False, groups=1, op='+', n_reps=1, **kwargs):
-        super().__init__()
-
+    def __init__(self, inputs=None, layout='cnacn', filters='same', kernel_size=3, strides=1,
+                 downsample=False, bottleneck=False, attention_mode=False, groups=1, op='+a', n_reps=1, **kwargs):
```

Comment:
```suggestion
                 downsample=False, bottleneck=False, attention=None, groups=1, op='+a', n_reps=1, **kwargs):
```
There is no docstring for `attention`